Easydict/NewApp/Feature/Shortcut/Shortcut+Default.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Shortcut+Default.swift
+//  Easydict
+//
+//  Created by Sharker on 2024/2/5.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Defaults
+import Magnet
+
+extension Shortcut {
+    // set defalut for app shortcut
+    func setDefaultForAppShortcut() {
+        Defaults[.clearInputShortcut] = KeyCombo(QWERTYKeyCode: 40, cocoaModifiers: .command)
```

Comment:
```suggestion
        Defaults[.clearInputShortcut] = KeyCombo(key: .k, cocoaModifiers: .command)
```

Can we improve this code like this?

Reply:
okay, that creates KeyCombo clearly